data/bestiary/creatures-b3.json

```diff
@@ -33489,6 +33492,33 @@
 					]
 				}
 			],
+			"spellcasting": [
+				{
+					"name": "Monk Ki",
+					"type": "Focus",
+					"fp": 3,
+					"DC": 24,
+					"entries": [
```

Comment:
I thought we dropped `entries`? 🤣

Reply:
Oops, staged the other version as well accidentally 🤣. Thanks for the catch!